src/redux/reducers/createRouteReducer.js

```diff
@@ -250,6 +253,21 @@ export default function reducer (state = initialState, action = {}) {
         case types.ROUTE_FINISH_SUCCESS:
             return state.set('finito', true);
 
+        case types.ROUTES_CREATE_TOGGLE_DRAFT:
```

Comment:
IMO the way we update isPrivate / isActive / isDraft should be exactly the same, i.e same action names, same actionType name-structure. Also cant we just have a 
{
   updatingRoute: {
      error: 
      isUpdating:
 }
}

or something in the routes reducer, that is used for all three states. It's basicly just an update on a property